src/gwt/www/templates/encrypted-sign-in.htm

```diff
@@ -167,6 +167,10 @@ <h2 id="caption">Sign in to RStudio</h2>
          <label for="username">Username:</label><br />
          <input type='text' 
                 name='username' 
+                autocomplete='off'
```

Comment:
When you tested this change locally, did iOS still offer to fill in the username/password? While I think turning off spelling checks makes sense, we don't want to disable the browser's password manager. (I suspect that `autocomplete` doesn't affect the password manager, but you probably know for sure!)

Reply:
I'll go through that today and report back (this came up on @mrshrbrmstr's new iPad and I was tasked with "make this go away"). It should not prevent folks using the latest iOS 12.x password API but I'll make sure and let you know.

Reply:
can confirm that on on all macOS-capable browsers and on iOS that you can still use password managers (and the new universal password manager on iOS) to fill in forms.

Reply:
Great, thanks for validating that!